### ISSUE TITLE
backspace not working in txt file editor on Android web browser (any)

### ISSUE THREAD

**Issue by gardiol** [owncloud/core#22687]
### Steps to reproduce
1. create new TXT file
2. try to edit it on Android trough web browser
3. try to delete characters with backspace key

### Expected behaviour
You can edit it and delete characters by hitting backspace

### Actual behaviour
Nothing, like not pressing backspace at all

### Client configuration
**Browser:**
Tested with Firefox for Android Chrome and stock Samsung browser

**Operating system:**
Android, tested on 4.4.2

===========================
In other words, the txt file editor on the web page, when used on Android, does not allow to delete any cnaracter. You can write, but not delete. The "note" app, for example, or the password app, they work just fine.

**Comment by gardiol** [comment]
Sorry trying to post from cell phone... Did.not worm very well. I will try to fix this post later on today.

**Comment by LukasReschke** [comment]
Please reopen at https://github.com/owncloud/files_texteditor/

**Comment by gardiol** [comment]
Ok, here: https://github.com/owncloud/files_texteditor/issues/165
